doc/guide/Makefile-guide.am

```diff
@@ -38,9 +38,12 @@ CLEANFILES += \
 	*.tmp \
 	$(NULL)
 
-doc/guide/html/index.html: $(GUIDE_DOCBOOK) $(GUIDE_INCLUDES) $(man_MANS) $(GUIDE_STATIC) $(GUIDE_XSLT)
+doc/guide/html: $(GUIDE_DOCBOOK) $(GUIDE_INCLUDES) $(man_MANS) $(GUIDE_STATIC) $(GUIDE_XSLT)
 	$(AM_V_GEN) mkdir -p doc/guide/html/ && rm -rf doc/guide/html/* && \
-		cp $(srcdir)/doc/guide/static/* doc/guide/html/ && \
 		$(XMLTO) html -m $(srcdir)/doc/guide/gtk-doc.xsl -o doc/guide/html/ \
 			--searchpath $(abs_builddir):$(abs_srcdir):$(abs_builddir)/doc/guide \
-			$(srcdir)/$(GUIDE_DOCBOOK)
+			$(srcdir)/$(GUIDE_DOCBOOK) && \
+		cp $(srcdir)/doc/guide/static/* doc/guide/html/
+
+doc/guide/html/index.html:
```

Comment:
Why is this necessary? Shouldn't we just update install-doc-local to be:

```
install-doc-local: doc/guide/html
```

and remove these lines?